internal/dataplane/sendconfig/config_getter.go

```diff
@@ -16,7 +16,13 @@ const (
 	WellKnownInitialHash = "00000000000000000000000000000000"
 )
 
-type ConfigurationChangeDetector interface {
+type ConfigurationGetter interface {
```

Comment:
Hmm, I'm not sure if the name of the interface really matches what this is responsible for. The name claims it gets the configuration while it is capable of:
- getting the current status 
- detecting configuration change 

IMHO neither of these matches the name which suggests to me that maybe they should not belong to a single interface.

Reply:
Since I removed the `GetCurrentStatus` method, we don't need to perform any changes here. Renamed.